### PR TITLE
modulesync 2.9.0

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 1.9.4

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@
   docker_sets:
   - set: centos6-64
   - set: centos7-64
-  - set: debian7-64
```
Comment (dhoppe) [inline .sync.yml:6]: @bastelfreak Did you missed to add ```debian9-64``` instead?

File: spec/spec_helper_acceptance.rb [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
-      on host, puppet('module', 'install', 'camptocamp-puppetserver')
-      on host, puppet('module', 'install', 'puppetlabs-inifile')
-      on host, puppet('module', 'install', 'puppetlabs-puppetserver_gem')
+  hosts.each do |host|
+    if host[:platform] =~ %r{el-7-x86_64} && host[:hypervisor] =~ %r{docker}
+      on(host, "sed -i '/nodocs/d' /etc/yum.conf")
     end
+    install_module_from_forge('puppet-puppetserver', '>= 3.0.0 < 4.0.0')
+    install_module_from_forge('puppetlabs-inifile', '>= 2.3.0 < 3.0.0')
+    install_module_from_forge('puppetlabs-puppetserver_gem', '>= 1.0.0 < 2.0.0')
```
Comment (ekohl) [inline spec/spec_helper_acceptance.rb:20]: This is already listed in `metadata.json`

File: spec/spec_helper_acceptance.rb [hunk 3]
```diff
@@ -1,27 +1,22 @@
...
-    hosts.each do |host|
-      on host, puppet('module', 'install', 'puppetlabs-stdlib')
-
-      # needed to test reporting
-      on host, puppet('module', 'install', 'camptocamp-puppetserver')
-      on host, puppet('module', 'install', 'puppetlabs-inifile')
-      on host, puppet('module', 'install', 'puppetlabs-puppetserver_gem')
+  hosts.each do |host|
+    if host[:platform] =~ %r{el-7-x86_64} && host[:hypervisor] =~ %r{docker}
+      on(host, "sed -i '/nodocs/d' /etc/yum.conf")
```
Comment (ekohl) [inline spec/spec_helper_acceptance.rb:16]: I do wonder why this is needed.

**Comment by ekohl** [comment]
Closing per https://github.com/voxpupuli/puppet-report_hipchat/issues/44

**Pull request closed without merging**